stereo_image_proc/launch/stereo_image_proc.launch.py

```diff
@@ -224,7 +226,15 @@ def generate_launch_description():
                     PythonLaunchDescriptionSource([
                         FindPackageShare('image_proc'), '/launch/image_proc.launch.py'
                     ]),
-                    launch_arguments={'container': LaunchConfiguration('container')}.items()
+                    launch_arguments={
+                        'container': PathJoinSubstitution([
+                            PythonExpression([
+                                '"/".join("',
+                                LaunchConfiguration('ros_namespace'),
+                                '".split("/")[:-1])'
```

Comment:
I presume this is to remove the "left_namespace" part? What if we instead prepend the ROS namespace at L220? I think this might make the change a bit simpler.